tests/Microsoft.Identity.Test.Unit/CoreTests/InstanceTests/GenericAuthorityTests.cs

```diff
@@ -333,6 +333,50 @@ public async Task BadOidcResponse_ThrowsException_Async(string badOidcResponseTy
             }
         }
 
+        [TestMethod]
+        public async Task OidcIssuerValidation_AcceptsDifferentPath_Async()
+        {
+            using (var httpManager = new MockHttpManager())
+            {
+                string authority = "https://demo.duendesoftware.com/organizations/2.0/";
```

Comment:
Let's use https://login.microsoftonline.com/organizations/v2.0 like the Id.Web bug in the test.

Reply:
Adjusted in the latest commit. Since the issue is probably only going to happen when using a Microsoft host, the test now uses "login.microsoftonline.com" instead of the custom domain.